src/.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <!-- Required elements-->
+        <id>SwedbankPay.Sdk</id>
+        <description>A client library and SDK for Swedbank Pay's RESTful API Platform</description>
+        <authors>Swedbank Pay</authors>
+        <version>1.1.1</version>
+
+        <!-- Optional elements -->
+        <projectUrl>https://github.com/SwedbankPay/swedbank-pay-sdk-dotnet</projectUrl>
+        <license type="expression">Apache-2.0</license>
+        <icon>icon.png</icon>
+        <copyright>Swedbank Pay</copyright>
+        <tags>SwedbankPay sdk payments payment client api rest restful</tags>
+        <language>en-US</language>
+    </metadata>
+    <files>
+        <file src="../icon.png" target="" />
```

Comment:
The `icon` element above references `./icon.png`, while this `file` element references `../icon.png`. Which one is correct? And is it necessary to explicitly include `icon.png` when it is already referenced with `<icon/>`?

Reply:
Yes, it's necessary. https://docs.microsoft.com/en-us/nuget/reference/nuspec#icon